module/motion/QuinticWalk/data/plotjuggler/layout.xml

```diff
@@ -0,0 +1,193 @@
+   <scripts/>
+  </plugin>
+  <plugin ID="CSV Exporter"/>
+ </Plugins>
+ <!-- - - - - - - - - - - - - - - -->
+ <previouslyLoaded_Datafiles/>
+ <!-- - - - - - - - - - - - - - - -->
+ <customMathEquations/>
+ <snippets/>
+ <!-- - - - - - - - - - - - - - - -->
+</root>
+
```

Comment:
What's this?

Reply:
Its a chart layout for plot juggler which saves you having to recreate one every time you want to debug. Not sure if we should store these here.